siteimprove/includes/class-siteimprove.php

```diff
@@ -145,8 +145,10 @@ private function define_admin_hooks() {
 		$this->loader->add_action( 'siteimprove_before_settings_form', $plugin_admin, 'siteimprove_before_settings_form' );
 
 		// Siteimprove Actions.
-		if ( ! isset( $_GET['si_preview'] ) || '0' === $_GET['si_preview'] ) {
-			$this->loader->add_action( 'admin_init', $plugin_admin, 'siteimprove_init' );
+		$this->loader->add_action( 'admin_init', $plugin_admin, 'siteimprove_init' );
```

Comment:
I think the init should also be part of the else branch, as it was before the changes.